Composer/packages/tools/language-servers/language-generation/src/utils.ts

```diff
@@ -142,3 +142,18 @@ export const cardPropDict = {
   Attachment: ['contenttype', 'content'],
   Others: ['type', 'name', 'value'],
 };
+
+export function getCurrLine(document: TextDocument, lineCount: number, line: number) {
```

Comment:
you dont need to pass lineCount, document has it.

Better name for function is getLineByIndex

Reply:
Also, can you just ```document.getText().split('\n')``` to get the lines?

Reply:
Fixed the signature and naming. Also use `document.getText().split(/\r?\n/g)` to get line content.